pkg/actors/reminders/reminders.go

```diff
@@ -854,6 +866,15 @@ func (r *reminders) startReminder(reminder *internal.Reminder, stopChannel chan
 				nextTimer = nil
 				return
 			}
+			track, gErr := r.getReminderTrack(context.TODO(), reminderKey)
```

Comment:
Why getting the track again? Don't we have it already at the start of the method?

Reply:
Yeah, correct. Deleted it.